apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -28,11 +28,10 @@ import NavItem from '../nav-item';
 import { Post } from '../../types';
 import './style.scss';
 
-const Button = ( {
-	children,
-	...rest
-}: OriginalButton.Props & { icon?: any; iconSize?: number } ) => (
-	<OriginalButton { ...rest }>{ children }</OriginalButton>
+const Button = forwardRef(
+	( { children, ...rest }: OriginalButton.Props & { icon?: any; iconSize?: number } ) => (
+		<OriginalButton { ...rest }>{ children }</OriginalButton>
+	)
```

Comment:
@p-jackson  Per your comment in slack:
> In your call to forwardRef, it looks like the second parameter isn’t being forwarded to <OriginalButton> ? It’s messy unfortunately. Apologies for the messy TS junk needed to define those missing props.

I'm not sure I follow you.  What is the "second parameter"?  And I guess what is the "first parameter" would be another question.  😆 

I see an example of usage [here](https://github.com/Automattic/wp-calypso/blob/720b5dad4f5217adc9c1de3e2d706b56f6bb7cf9/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-item/index.tsx#L24).   As far as I understand, I'm doing the same .  But indeed, I might be very confused. :)